index.html

```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./Styles/styles.css" type="text/css">
+    <script src="https://kit.fontawesome.com/c2d0b29889.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <title>Daniel Yerimah's Capstone Project</title>
+</head>
+<body>
+    <header>
+    <div class="topnav">
+        <ul>
+            <li><a href="#" class="fa fa-facebook"></a></li>
+            <li><a href="#" class="fa fa-twitter"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+    </div>
+    <div class="container">
+    <div class="navbar">
+        <div class="logo">
+            <a href="index.html"><img src="./images/newlogo.png" alt="African Tech Summit Logo"></a>
+        </div>
+        <nav>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li class="border-about"><a href="#">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </div>
+    </div>
+        <div class="bars" id="hambug">
+        <div class="hambuger"></div>
+        <div class="hambuger"></div>
+        <div class="hambuger"></div>
+        </div>
+
+    <div class="toggle-menu">
+            <div><img src="./images/closetoggle.png" alt="close toggle" id="togclose" ></div>
```

Comment:
Please make sure the X closing button is showing correctly. However, the functionality is working well 👍👍 .  As you can see in the image below
![image](https://user-images.githubusercontent.com/60292761/148426806-ff737f6e-40ec-47bd-aef2-8c5e3746db2f.png)